worldedit-sponge/src/main/java/com/sk89q/worldedit/sponge/SpongePlatform.java

```diff
@@ -197,15 +177,13 @@ public String getPlatformVersion() {
 
     @Override
     public Set<SideEffect> getSupportedSideEffects() {
-        return ImmutableSet.of();
+        return ImmutableSet.of(
+            SideEffect.UPDATE, SideEffect.ENTITY_AI, SideEffect.LIGHTING, SideEffect.NEIGHBORS
```

Comment:
No `VALIDATION`?

Reply:
No, Sponge does not support it as a separate side effect